package.json

```diff
@@ -1,20 +1,25 @@
 {
-  "name": "plugin-check",
-  "license": "GPL-2.0-or-later",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/10up/plugin-check.git"
-  },
-  "devDependencies": {
-    "@wordpress/env": "^5.7.0",
-    "@wordpress/scripts": "^24.6.0"
-  },
-  "scripts": {
-    "wp-env": "wp-env",
-    "prelint-php": "wp-env run composer 'install --no-interaction'",
-    "lint-php": "wp-env run composer run-script lint",
-    "pretest-php": "wp-env run composer 'install --no-interaction'",
-    "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/plugin-check/phpunit.xml.dist --verbose'",
-    "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/plugin-check/tests/multisite.xml --verbose'"
-  }
-}
\ No newline at end of file
+	"name": "plugin-check",
+	"version": "1.0.0",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/10up/plugin-check.git"
+	},
+	"devDependencies": {
+		"@wordpress/env": "^5.7.0",
+		"@wordpress/scripts": "^24.6.0"
+	},
+	"scripts": {
+		"wp-env": "wp-env",
+		"format": "wp-scripts format",
+		"lint:js": "wp-scripts lint-js",
+		"lint:md:docs": "wp-scripts lint-md-docs",
+		"packages-update": "wp-scripts packages-update",
```

Comment:
What do these do? Do we really need them? Seems a bit beyond the scope of this issue.

Reply:
I can remove it.